doc/basics/compilation.md

```diff
@@ -74,6 +74,12 @@ $ cmake ..
 $ make
 ~~~
 
+Finally, the install target will copy all the radium related library in the same place, usefull for App compilation 
+
+~~~bash
+$ make install
+~~~
+
 To run on Mac OS X, one may need to set `DYLD_LIBRARY_PATH` to `path/to/Bundle-*/3rdPartyLibraries/lib`
```

Comment:
to be removed. No more needed.

Reply:
Could you provide a suggestion, because it not clear what I should remove

Reply:
```suggestion
```

Reply:
Or just confirm thiss one ;) I will commit if its ok.

Reply:
OK, you can confirm. I've not the permission to push to the repo.